templates/web/angus/post_header_extra.html

```diff
@@ -0,0 +1,14 @@
+<div class="wrapper wrapper__breadcrumb">
```

Comment:
This was me getting confused with the purposes of existing CSS classes  - I didn't need to use `.wrapper` at all.